operator-framework/src/main/java/com/github/containersolutions/operator/processing/EventDispatcher.java

```diff
@@ -75,14 +91,14 @@ private boolean hasDefaultFinalizer(CustomResource resource) {
     }
 
     private void removeDefaultFinalizer(CustomResource resource) {
+        log.debug("Removing finalizer on {}: {}", resource);
```

Comment:
If we log entire resource it becomes very verbose. Is it possible to use `trace` whenever we want to log entire resource.

Reply:
> If we log entire resource it becomes very verbose. Is it possible to use `trace` whenever we want to log entire resource.

You are right, could you pls create an issue to sanitize logs?